modules/rest/src/main/java/com/enonic/xp/app/contentstudio/rest/resource/content/ContentPathHelper.java

```diff
@@ -0,0 +1,27 @@
+package com.enonic.xp.app.contentstudio.rest.resource.content;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.enonic.xp.content.ContentPath;
+
+public class ContentPathHelper
```

Comment:
use `ContentPath.from` instead

Reply:
Won't work, sole purpose of this helper is to replace a function from ContentPath -  asRelative() that was removed, and it was meant to set 'absolute' param of ContentPath to 'false', but 'ContentPath.from' copies the value of 'absolute' from parent, without possibility to override it. Also it is only possible to create a builder class for ContentPath with empty constructor, would be nice to have a builder 'ContentPath.create( final ContentPath source)', then the helper would lool like:
```

public static ContentPath toRelative( final ContentPath source )
    {
        return new ContentPath.Builder(source).absolute( false ).build();
    }
```

Reply:
it was removed because relative content paths are evil
what is wrong with
`public static ContentPath from( final ContentPath parent, final String name )` ?

And why not using builder where it is still possible to configure absolute/relative.

For sure - constructing content path via string is the worst option